types/queries/update_word.sql

```diff
@@ -8,6 +8,12 @@ update word set
         case
             when $3::text[] != '{}' then $3[1]
             else commentary
+        end,
+
+    english_translation =
```

Comment:
Should be called english_gloss as defined in types/migrations/20220205042039_init.sql

Reply:
i also renamed the instance of english_translation below like this 
```
    english_gloss=
        case
            when $4::text[] != '{}' and $4[1] is not null then $4[1]
            else english_gloss
```